### PR TITLE
PSV-8: FE init

### PR DESCRIPTION
**Pull request by blam92**
PSV-8: https://patent-visualization.atlassian.net/jira/software/projects/PSV/boards/1?selectedIssue=PSV-8

I do not consider my story done, but wanted to get the first set of changes in to make the PR manageable + unblock anyone that wants to start working on UI functionality for a given page.

*Summary*
- Added dependencies Ant Design and React Router
- Initial directory structure for components
- Moved App.js into its own dir EntryPoint
- Basic routing and components for Upload Patents and Search apge

Some pending items NOT covered in this PR:
- Clean up strings
- Simple tests for these presentational components
- Setup some sort of ESLint + precommit hook
- Better CSS. I did introduce BEM as a pattern to follow that is easy and gets the job done http://getbem.com/naming/

Search screen:

*MANDATORY GIF FOR EVERY PR*

### CODE REVIEW

**Comment by blam92** [comment]
with dropdown open

File: src/components/Search/Search.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+        name="sequence"
+        className="search-container__input"
+        rules={[{ required: false }]}
+      >
+        <Input />
+      </Form.Item>
+
+      <Dropdown overlay={menu}>
+        <Button className="search-container__dropdown">
+            Select Protein <DownOutlined />
```
Comment (kaliweh) [inline src/components/Search/Search.js:51]: I would change this dropdown in future commits to a simple textbox that calls another search. Once we have more than 10 proteins, the dropdown would become unusable🙌

File: src/components/Search/Search.js [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+  return (
+    <Form
+      name="basic"
+      className="search-container"
+      initialValues={{ remember: true }}
+      onFinish={onFinish}
+      onFinishFailed={onFinishFailed}
+    >
+      <Form.Item
+        label="Antibody Sequence"
```
Comment (kaliweh) [inline src/components/Search/Search.js:41]: Antigen or epitope sequence.

**Review by kaliweh — APPROVED** [review]
⛴  it

File: src/components/Search/Search.js (hunk 1 above)
Comment (blam92) [reply src/components/Search/Search.js:51]: good point. Will do

**Pull request merged**